sonar-plugin-api/src/main/java/org/sonar/api/config/Settings.java

```diff
@@ -127,6 +127,7 @@ protected Settings(PropertyDefinitions definitions, Encryption encryption) {
    */
   public abstract Map<String, String> getProperties();
 
+  // FIXME scope to be replaced by "protected" as soon as not used by JRubyFacade
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1134)